thirdparty/fetch-thirdparty-deps.ps1

```diff
@@ -130,4 +130,19 @@ if (!(Test-Path "Eluant.dll"))
 	(New-Object System.Net.WebClient).DownloadFile("https://github.com/OpenRA/Eluant/releases/download/20140425/Eluant.dll", $target)
 }
 
+if ((Test-Path "GeoLite2-Country.mmdb"))
+{
+	$lastWrite = (get-item "GeoLite2-Country.mmdb").LastWriteTime
+	$timespan = new-timespan -days 30
+}
+
+if (!(Test-Path "GeoLite2-Country.mmdb") -Or (((get-date) - $lastWrite) -gt $timespan))
```

Comment:
You should be able to move the lastWrite and timespan inline, and avoid the extra block and variables above.